testcharms/charms/simple-resolve/metadata.yaml

```diff
@@ -0,0 +1,15 @@
+name: simple-resolve
+maintainer: Christopher Lee <chris.mclachlan-lee@canonical.com>
+summary: Purposfully error on install hook for testing resolve command.
+description: |
+  A testing charm that errors once on install hook then succeeds on the next run.
+categories:
+  - misc
+series:
```

Comment:
Juju is soon to only support jammy and focal, so the rest of these should probably be dropped

Reply:
agreed